### PR TITLE
feat(controller): treat "master" as latest version

### PR DESCRIPTION
**Pull request by lobshunter**
Signed-off-by: lobshunter <pengyu@pingcap.com>

### What problem does this PR solve?
This PR adds `master` to latest version list, so images with that tag can be used(mainly used in testing).

### Code changes

- [x] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [x] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
Treat "master" as latest version of TiDB components too
```

### CODE REVIEW

File: pkg/util/cmpver/cmpver.go [hunk 1]
```diff
@@ -130,8 +131,13 @@ func validateOperation(op Operation) error {
 }
 
 func isLatest(version string) bool {
-	switch version {
-	case "latest", "nightly":
+	switch {
```
Comment (KanShiori) [inline pkg/util/cmpver/cmpver.go:134]: please update above comments.

> // Latest or nightly version is larger than any version

**Comment by KanShiori** [comment]
@csuzhangxc PTAL

**Review by csuzhangxc — APPROVED** [review]

File: pkg/util/cmpver/cmpver.go (hunk 1 above)
Comment (lobshunter) [reply pkg/util/cmpver/cmpver.go:134]: Updated in 793aaa46222d8d980524d9e95a834c6f583e87f0

**Pull request merged**